Xwt.XamMac/Xwt.Mac/PanedBackend.cs

```diff
@@ -118,25 +120,83 @@ public override void ReplaceChild (NSView oldChild, NSView newChild)
 				Widget.AddSubview (view2);
 		}
 
-		public double Position {
-			get {
-				return 0;
+		private double position;
+		public double Position
+		{
+			get
+			{
+				return position;
 			}
-			set {
+			set
+			{
+				position = value;
+				this.DidResizeSubviews();
 			}
 		}
 		#endregion
 	}
-	
-	class CustomSplitView: NSSplitView, IViewObject
+
+	class CustomSplitView : NSSplitView, IViewObject
 	{
-		public NSView View {
-			get {
+		readonly double initDividerPosition = 220;
```

Comment:
Can we just have a readonly nfloat here?

Reply:
Fixed.